<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><head><title>R: Summaries and Population Structure Analyses of Haplotypic and
Genotypic Data</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<link rel="stylesheet" type="text/css" href="R.css">
</head><body>
<h1> Summaries and Population Structure Analyses of Haplotypic and
Genotypic Data
<img class="toplogo" src="../../../doc/html/logo.jpg" alt="[R logo]">
</h1>
<hr>
<div align="center">
<a href="../../../doc/html/packages.html"><img src="../../../doc/html/left.jpg" alt="[Up]" width="30" height="30" border="0"></a>
<a href="../../../doc/html/index.html"><img src="../../../doc/html/up.jpg" alt="[Top]" width="30" height="30" border="0"></a>
</div><h2>Documentation for package &lsquo;strataG&rsquo; version 1.0</h2>

<ul><li><a href="../DESCRIPTION">DESCRIPTION file</a>.</li>
</ul>

<h2>Help Pages</h2>


<p align="center">
<a href="# "> </a>
<a href="#A">A</a>
<a href="#B">B</a>
<a href="#C">C</a>
<a href="#D">D</a>
<a href="#E">E</a>
<a href="#F">F</a>
<a href="#G">G</a>
<a href="#H">H</a>
<a href="#I">I</a>
<a href="#J">J</a>
<a href="#L">L</a>
<a href="#M">M</a>
<a href="#N">N</a>
<a href="#O">O</a>
<a href="#P">P</a>
<a href="#R">R</a>
<a href="#S">S</a>
<a href="#T">T</a>
<a href="#V">V</a>
<a href="#W">W</a>
</p>

<table width="100%">
<tr><td width="25%"><a href="strataG-package.html">strataG-package</a></td>
<td>Summaries and population structure analyses of haplotypic and genotypic data</td></tr>
</table>

<h2><a name="A">-- A --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="align.clustal.html">align.clustal</a></td>
<td>CLUSTAL Alignment</td></tr>
<tr><td width="25%"><a href="align.folder.html">align.folder</a></td>
<td>Align Sequences in a Folder</td></tr>
<tr><td width="25%"><a href="align.mafft.html">align.mafft</a></td>
<td>MAFFT Alignment</td></tr>
<tr><td width="25%"><a href="allele.freq.format.html">allele.freq.format</a></td>
<td>Compiles and Formats Allele Frequencies</td></tr>
<tr><td width="25%"><a href="allele.freqs.html">allele.freqs</a></td>
<td>Allele Frequencies</td></tr>
<tr><td width="25%"><a href="allele.split.html">allele.split</a></td>
<td>Split Alleles For Diploid Data</td></tr>
<tr><td width="25%"><a href="allelic.richness.html">allelic.richness</a></td>
<td>Allelic Richness</td></tr>
<tr><td width="25%"><a href="pop.diff.tests.html">analysis.list</a></td>
<td>Population Differentiation Tests</td></tr>
<tr><td width="25%"><a href="as.data.frame.gtypes.html">as.data.frame.gtypes</a></td>
<td>Coerce to a Data Frame</td></tr>
<tr><td width="25%"><a href="as.dna.seq.html">as.dna.seq</a></td>
<td>Format List of DNA Sequences</td></tr>
<tr><td width="25%"><a href="as.matrix.gtypes.html">as.matrix.gtypes</a></td>
<td>Coerce to a Matrix</td></tr>
</table>

<h2><a name="B">-- B --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="base.freqs.html">base.freqs</a></td>
<td>Calculate Nucleotide Frequencies</td></tr>
<tr><td width="25%"><a href="bowhead.snp.position.html">bowhead.snp.position</a></td>
<td>Bowhead Whale SNP Genotype Groups</td></tr>
<tr><td width="25%"><a href="bowhead.snps.html">bowhead.snps</a></td>
<td>Bowhead Whale SNP Genotypes</td></tr>
</table>

<h2><a name="C">-- C --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="gtypes.html">check.gtypes</a></td>
<td>Create a gtypes object</td></tr>
<tr><td width="25%"><a href="clumpp.run.html">clumpp</a></td>
<td>Run CLUMPP</td></tr>
<tr><td width="25%"><a href="clumpp.run.html">clumpp.run</a></td>
<td>Run CLUMPP</td></tr>
<tr><td width="25%"><a href="create.consensus.html">create.consensus</a></td>
<td>Create Consensus Sequence</td></tr>
</table>

<h2><a name="D">-- D --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="nei.Da.html">Da</a></td>
<td>Nei's Da</td></tr>
<tr><td width="25%"><a href="decode.html">decode</a></td>
<td>Decode 'gtypes'</td></tr>
<tr><td width="25%"><a href="decode.html">decode.loci</a></td>
<td>Decode 'gtypes'</td></tr>
<tr><td width="25%"><a href="decode.html">decode.sequences</a></td>
<td>Decode 'gtypes'</td></tr>
<tr><td width="25%"><a href="decode.html">decode.strata</a></td>
<td>Decode 'gtypes'</td></tr>
<tr><td width="25%"><a href="diversity.html">diversity</a></td>
<td>Unbiased Estimate of Diversity</td></tr>
<tr><td width="25%"><a href="dolph.haps.html">dolph.haps</a></td>
<td>Dolphin mtDNA Haplotype Sequences</td></tr>
<tr><td width="25%"><a href="dolph.msats.html">dolph.msats</a></td>
<td>Dolphin Microsatellite Genotypes</td></tr>
<tr><td width="25%"><a href="dolph.seqs.html">dolph.seqs</a></td>
<td>Dolphin mtDNA Sequences</td></tr>
<tr><td width="25%"><a href="dolph.strata.html">dolph.strata</a></td>
<td>Dolphin Genetic Stratification and hHplotypes</td></tr>
</table>

<h2><a name="E">-- E --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="exptd.het.html">exptd.het</a></td>
<td>Expected Heterozygosity</td></tr>
</table>

<h2><a name="F">-- F --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="fastsimcoal.run.html">fastsimcoal.run</a></td>
<td>Run FASTSIMCOAL</td></tr>
<tr><td width="25%"><a href="fastsimcoal.run.html">fastsimcoal.write</a></td>
<td>Run FASTSIMCOAL</td></tr>
<tr><td width="25%"><a href="fisher.method.p.html">fisher.method.p</a></td>
<td>Fisher's Method P-value</td></tr>
<tr><td width="25%"><a href="fixed.differences.html">fixed.differences</a></td>
<td>Fixed Base Pair Differences Between Strata</td></tr>
<tr><td width="25%"><a href="fixed.sites.html">fixed.sites</a></td>
<td>Fixed Sites Among Sequences</td></tr>
<tr><td width="25%"><a href="fst.to.nm.html">fst.to.nm</a></td>
<td>Calculate Nm from Fst</td></tr>
</table>

<h2><a name="G">-- G --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="gelato.plot.html">gelato.plot</a></td>
<td>Plot GELATo results</td></tr>
<tr><td width="25%"><a href="gelato.run.html">gelato.run</a></td>
<td>GELATo - Group ExcLusion and Assignment Test</td></tr>
<tr><td width="25%"><a href="gen.data.from.freq.html">gen.data.from.freq</a></td>
<td>Convert Haplotype Frequency Matrices</td></tr>
<tr><td width="25%"><a href="genepop.run.html">genepop.run</a></td>
<td>Run GENEPOP</td></tr>
<tr><td width="25%"><a href="genepop.run.html">genepop.write</a></td>
<td>Run GENEPOP</td></tr>
<tr><td width="25%"><a href="gtype.struct.func.html">gtype.struct.func</a></td>
<td>Population Structure Statistic Functions</td></tr>
<tr><td width="25%"><a href="gtype.struct.func.html">gtype.struct.stat</a></td>
<td>Population Structure Statistic Functions</td></tr>
<tr><td width="25%"><a href="gtypes.html">gtypes</a></td>
<td>Create a gtypes object</td></tr>
<tr><td width="25%"><a href="gtypes.html">gtypes.character</a></td>
<td>Create a gtypes object</td></tr>
<tr><td width="25%"><a href="gtypes.html">gtypes.default</a></td>
<td>Create a gtypes object</td></tr>
<tr><td width="25%"><a href="gtypes.html">gtypes.list</a></td>
<td>Create a gtypes object</td></tr>
</table>

<h2><a name="H">-- H --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="h.stats.html">h.stats</a></td>
<td>Estimate Fixation Indices</td></tr>
<tr><td width="25%"><a href="hap.freqs.html">hap.freqs</a></td>
<td>Haplotype Frequencies</td></tr>
<tr><td width="25%"><a href="haplotypic.diversity.html">haplotypic.diversity</a></td>
<td>Haplotypic Diversity</td></tr>
<tr><td width="25%"><a href="hwe.genepop.html">hwe.genepop</a></td>
<td>Hardy-Weinberg Equilibrium</td></tr>
</table>

<h2><a name="I">-- I --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="as.dna.seq.html">is.aligned.seq</a></td>
<td>Format List of DNA Sequences</td></tr>
<tr><td width="25%"><a href="is.diploid.html">is.diploid</a></td>
<td>Check Ploidy</td></tr>
<tr><td width="25%"><a href="as.dna.seq.html">is.dna.seq</a></td>
<td>Format List of DNA Sequences</td></tr>
<tr><td width="25%"><a href="is.gtype.struct.func.html">is.gtype.struct.func</a></td>
<td>Check Population Structure Function</td></tr>
<tr><td width="25%"><a href="gtypes.html">is.gtypes</a></td>
<td>Create a gtypes object</td></tr>
<tr><td width="25%"><a href="is.diploid.html">is.haploid</a></td>
<td>Check Ploidy</td></tr>
<tr><td width="25%"><a href="ti.tv.ratio.html">is.ti.tv</a></td>
<td>Transition / Transversion Ratio</td></tr>
<tr><td width="25%"><a href="iupac.code.html">iupac.code</a></td>
<td>IUPAC Code</td></tr>
<tr><td width="25%"><a href="iupac.mat.html">iupac.mat</a></td>
<td>IUPAC Codes</td></tr>
</table>

<h2><a name="J">-- J --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="jack.hwe.html">jack.hwe</a></td>
<td>Hardy-Weinberg Equlibrium Jackknife</td></tr>
<tr><td width="25%"><a href="jack.hwe.html">jack.influential</a></td>
<td>Hardy-Weinberg Equlibrium Jackknife</td></tr>
<tr><td width="25%"><a href="jmodeltest.run.html">jmodeltest.run</a></td>
<td>Run jModelTest</td></tr>
</table>

<h2><a name="L">-- L --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="label.haplotypes.html">label.haplotypes</a></td>
<td>Find and label haplotypes</td></tr>
<tr><td width="25%"><a href="label.haplotypes.html">label.haplotypes.default</a></td>
<td>Find and label haplotypes</td></tr>
<tr><td width="25%"><a href="label.haplotypes.html">label.haplotypes.gtypes</a></td>
<td>Find and label haplotypes</td></tr>
<tr><td width="25%"><a href="linkage.genepop.html">linkage.genepop</a></td>
<td>Linkage Disequlibrium</td></tr>
<tr><td width="25%"><a href="locus.names.html">locus.cols</a></td>
<td>Helper Functions to Extract Locus Information</td></tr>
<tr><td width="25%"><a href="locus.names.html">locus.names</a></td>
<td>Helper Functions to Extract Locus Information</td></tr>
</table>

<h2><a name="M">-- M --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="merge.gtypes.html">merge.gtypes</a></td>
<td>Merge gtypes</td></tr>
<tr><td width="25%"><a href="most.distant.sequences.html">most.distant.sequences</a></td>
<td>Most distant sequences</td></tr>
</table>

<h2><a name="N">-- N --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="na.omit.gtypes.html">na.omit.gtypes</a></td>
<td>Omit NAs in gtypes object</td></tr>
<tr><td width="25%"><a href="nei.Da.html">nei.Da</a></td>
<td>Nei's Da</td></tr>
<tr><td width="25%"><a href="nucleotide.divergence.html">nucleotide.divergence</a></td>
<td>Nucleotide Divergence</td></tr>
<tr><td width="25%"><a href="nucleotide.diversity.html">nucleotide.diversity</a></td>
<td>Nucleotide Diversity</td></tr>
<tr><td width="25%"><a href="num.alleles.html">num.alleles</a></td>
<td>Number of Alleles</td></tr>
<tr><td width="25%"><a href="num.alleles.shared.html">num.alleles.shared</a></td>
<td>Number of Shared Alleles</td></tr>
<tr><td width="25%"><a href="num.gens.eq.html">num.gens.eq</a></td>
<td>Calculate Number of Generations to Equilibrium</td></tr>
<tr><td width="25%"><a href="locus.names.html">num.loci</a></td>
<td>Helper Functions to Extract Locus Information</td></tr>
<tr><td width="25%"><a href="num.missing.html">num.missing</a></td>
<td>Calculate Number of Individuals With Missing Data</td></tr>
</table>

<h2><a name="O">-- O --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="obsvd.het.html">obsvd.het</a></td>
<td>Observed Heterozygosity</td></tr>
<tr><td width="25%"><a href="pop.diff.tests.html">overall.test</a></td>
<td>Population Differentiation Tests</td></tr>
</table>

<h2><a name="P">-- P --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="p.val.html">p.val</a></td>
<td>Permutation Test P-value</td></tr>
<tr><td width="25%"><a href="pairwise.shared.loci.html">pairwise.shared.loci</a></td>
<td>Calculate Pairwise Shared Loci</td></tr>
<tr><td width="25%"><a href="pop.diff.tests.html">pairwise.test</a></td>
<td>Population Differentiation Tests</td></tr>
<tr><td width="25%"><a href="pct.unique.haplotypes.html">pct.unique.haplotypes</a></td>
<td>Percent Unique Haplotypes</td></tr>
<tr><td width="25%"><a href="phase.run.html">phase</a></td>
<td>Run PHASE</td></tr>
<tr><td width="25%"><a href="phase.run.html">phase.filter</a></td>
<td>Run PHASE</td></tr>
<tr><td width="25%"><a href="phase.run.html">phase.posterior</a></td>
<td>Run PHASE</td></tr>
<tr><td width="25%"><a href="phase.run.html">phase.read.pair</a></td>
<td>Run PHASE</td></tr>
<tr><td width="25%"><a href="phase.run.html">phase.read.sample</a></td>
<td>Run PHASE</td></tr>
<tr><td width="25%"><a href="phase.run.html">phase.run</a></td>
<td>Run PHASE</td></tr>
<tr><td width="25%"><a href="phase.run.html">phase.write</a></td>
<td>Run PHASE</td></tr>
<tr><td width="25%"><a href="jack.hwe.html">plot.jack.influential</a></td>
<td>Hardy-Weinberg Equlibrium Jackknife</td></tr>
<tr><td width="25%"><a href="pop.diff.tests.html">pop.diff.tests</a></td>
<td>Population Differentiation Tests</td></tr>
<tr><td width="25%"><a href="print.gtypeSummary.html">print.gtypeSummary</a></td>
<td>Print gtypeSummary</td></tr>
</table>

<h2><a name="R">-- R --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="read.fasta.html">read.fasta</a></td>
<td>Read FASTA</td></tr>
<tr><td width="25%"><a href="read.gen.data.html">read.gen.data</a></td>
<td>Read Genetic Data</td></tr>
<tr><td width="25%"><a href="read.mega.html">read.mega</a></td>
<td>Read MEGA</td></tr>
<tr><td width="25%"><a href="relabel.sequences.html">relabel.sequences</a></td>
<td>Relabel Sequences</td></tr>
<tr><td width="25%"><a href="restratify.html">restratify</a></td>
<td>Restratify gtypes</td></tr>
</table>

<h2><a name="S">-- S --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="sequence.Da.html">sequence.Da</a></td>
<td>Calculate Da</td></tr>
<tr><td width="25%"><a href="sim.gamma.haps.html">sim.gamma.haps</a></td>
<td>Simulates Haplotypes</td></tr>
<tr><td width="25%"><a href="stat.chi2.html">stat.chi2</a></td>
<td>CHI-squared Analysis of Population Structure</td></tr>
<tr><td width="25%"><a href="stat.d.jost.html">stat.d.jost</a></td>
<td>Jost's D Estimate of Population Divergence</td></tr>
<tr><td width="25%"><a href="stat.fst.html">stat.fst</a></td>
<td>Fst Analysis of Population Structure</td></tr>
<tr><td width="25%"><a href="stat.fst.prime.html">stat.fst.prime</a></td>
<td>F'st analysis of population structure</td></tr>
<tr><td width="25%"><a href="stat.gst.html">stat.gst</a></td>
<td>Gst Analysis of Population Structure</td></tr>
<tr><td width="25%"><a href="stat.gst.dbl.prime.html">stat.gst.dbl.prime</a></td>
<td>G"st Analysis of Population Structure</td></tr>
<tr><td width="25%"><a href="stat.gst.prime.hedrick.html">stat.gst.prime.hedrick</a></td>
<td>Hedrick's G'st Analysis of Population Structure</td></tr>
<tr><td width="25%"><a href="stat.gst.prime.nei.html">stat.gst.prime.nei</a></td>
<td>Nei's G'st Analysis of Population Structure</td></tr>
<tr><td width="25%"><a href="stat.phist.html">stat.phist</a></td>
<td>PHIst Analysis of Population Structure</td></tr>
<tr><td width="25%"><a href="as.dna.seq.html">stopifnot.aligned</a></td>
<td>Format List of DNA Sequences</td></tr>
<tr><td width="25%"><a href="stopifnot.gtype.struct.func.html">stopifnot.gtype.struct.func</a></td>
<td>Check gtype Population Structure Function</td></tr>
<tr><td width="25%"><a href="gtypes.html">stopifnot.gtypes</a></td>
<td>Create a gtypes object</td></tr>
<tr><td width="25%"><a href="strata.split.html">strata.split</a></td>
<td>Split Strata</td></tr>
<tr><td width="25%"><a href="strataG-package.html">strataG</a></td>
<td>Summaries and population structure analyses of haplotypic and genotypic data</td></tr>
<tr><td width="25%"><a href="structure.run.html">structure</a></td>
<td>Run STRUCTURE</td></tr>
<tr><td width="25%"><a href="structure.evanno.html">structure.evanno</a></td>
<td>Run Evanno Method on STRUCTURE Results</td></tr>
<tr><td width="25%"><a href="structure.parse.q.mat.html">structure.parse.q.mat</a></td>
<td>Parse Q-Matrix Text</td></tr>
<tr><td width="25%"><a href="structure.plot.html">structure.plot</a></td>
<td>Plot STRUCTURE Results</td></tr>
<tr><td width="25%"><a href="structure.run.html">structure.read</a></td>
<td>Run STRUCTURE</td></tr>
<tr><td width="25%"><a href="structure.run.html">structure.run</a></td>
<td>Run STRUCTURE</td></tr>
<tr><td width="25%"><a href="structure.run.html">structure.write</a></td>
<td>Run STRUCTURE</td></tr>
<tr><td width="25%"><a href="subset.gtypes.html">subset.gtypes</a></td>
<td>Subset a gtypes object</td></tr>
<tr><td width="25%"><a href="summarize.loci.html">summarize.loci</a></td>
<td>Locus Summaries</td></tr>
<tr><td width="25%"><a href="summarize.sequences.html">summarize.sequences</a></td>
<td>DNA Sequence Summaries</td></tr>
<tr><td width="25%"><a href="summary.gtypes.html">summary.gtypes</a></td>
<td>Summarize gtypes object</td></tr>
</table>

<h2><a name="T">-- T --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="tajimas.d.html">tajimas.d</a></td>
<td>Calculate Tajima's D</td></tr>
<tr><td width="25%"><a href="theta.html">theta</a></td>
<td>Theta</td></tr>
<tr><td width="25%"><a href="ti.tv.mat.html">ti.tv.mat</a></td>
<td>Transition/Transversion matrix</td></tr>
<tr><td width="25%"><a href="ti.tv.ratio.html">ti.tv.ratio</a></td>
<td>Transition / Transversion Ratio</td></tr>
<tr><td width="25%"><a href="trim.ns.html">trim.ns</a></td>
<td>Trim N's From Sequences</td></tr>
</table>

<h2><a name="V">-- V --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="valid.iupac.codes.html">valid.iupac.codes</a></td>
<td>Valid IUPAC Codes</td></tr>
<tr><td width="25%"><a href="variable.sites.html">variable.sites</a></td>
<td>Variable Sites Among Sequences</td></tr>
</table>

<h2><a name="W">-- W --</a></h2>

<table width="100%">
<tr><td width="25%"><a href="wright.fst.html">wright.fst</a></td>
<td>Wright's Fst</td></tr>
<tr><td width="25%"><a href="write.arlequin.html">write.arlequin</a></td>
<td>Write Arlequin File</td></tr>
<tr><td width="25%"><a href="write.fasta.html">write.fasta</a></td>
<td>write.fasta</td></tr>
<tr><td width="25%"><a href="write.gtypes.html">write.gtypes</a></td>
<td>write.gtypes</td></tr>
<tr><td width="25%"><a href="write.MEGA.html">write.MEGA</a></td>
<td>Write MEGA File</td></tr>
<tr><td width="25%"><a href="write.snapp.nexus.html">write.snapp.nexus</a></td>
<td>Write NEXUS File for SNAPP</td></tr>
</table>
</body></html>
